ansible/roles/operator-pipeline/templates/openshift/pipelines/community-release-pipeline.yml

```diff
@@ -171,6 +171,60 @@ spec:
         - name: remove-matching-namespace-labels
           value: "true"
 
+    # set the GitHub status to success if the pipeline passes
+    - name: set-github-status-success
+      when:
+        - input: "$(tasks.status)"
+          operator: in
+          values: ["Succeeded", "Completed"]
+      taskRef:
+        name: set-github-status
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: git_repo_url
+          value: $(params.git_repo_url)
+        - name: commit_sha
+          value: $(params.git_commit)
+        - name: description
+          value: "Pull request for the operator was successfully merged"
```

Comment:
When a release pipeline is finished the MR is already merged for quite some time (actually the release pipeline is triggered by the merge event). So messages should say that an operator bundle has been successfully distributed. This also applies for failed case.

Reply:
Thanks, my bad, I forgot to realize its already after the merge event. I'll add more meaningful messages there